src/screens/claimables/shared/components/ClaimValueDisplay.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import RainbowCoinIcon from '@/components/coin-icon/RainbowCoinIcon';
+import { Bleed, Box, globalColors, Text, TextShadow, useColorMode } from '@/design-system';
+import { useTheme } from '@/theme';
+import { View } from 'react-native';
+import { IS_IOS } from '@/env';
+import { ShimmerAnimation } from '@/components/animations';
+
+export function ClaimValueDisplay({
+  label,
+  tokenIconUrl,
+  tokenSymbol,
+  chainId,
+}: {
+  label: string | undefined;
+  tokenIconUrl: string | undefined;
+  tokenSymbol: string | undefined;
+  chainId: number | undefined;
+}) {
```

Comment:
the reason why i explicitly typed `undefined` is because the props aren't optional (they should always be provided), but the values being passed in as props may be undefined

Reply:
I came around on the benefits on explicitly passing in undefined props, disregard.